dali/operators/generic/slice/slice_attr.h

```diff
@@ -47,7 +47,21 @@ class SliceAttr {
     }
   }
 
-  void ProcessArguments(MixedWorkspace &ws) {
+  void ProcessArguments(const DeviceWorkspace &ws) {
+    DALI_ENFORCE(ws.NumInput() == 3,
+      "Expected 3 inputs. Received: " + std::to_string(ws.NumInput()));
+    // slice args are CPU inputs
+    const auto &crop_anchor = ws.template Input<CPUBackend>(1);
```

Comment:
Maybe we should allow a GPU one as well?
@mzient ?

Reply:
If we decide to do so, it'd require some significant work, since now we do some checking/preprocessing on the slice arguments and we do that on the CPU